content/v0.75.0-rc1.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Introducing 0.75.0 Release Candidate 1"
+date = 2020-04-22
++++
+
+![Dark Forces (Win), Paku Paku (macOS), Settlers 2 (Linux)](/3games.png)
+
+Installation instructions:
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+This release is significant for several reasons:
+
+1. Our first stable build - from this point forward, builds named
+   0.75.x will only contain bugfixes.
+
+2. Start of the new development cycle for the next release - we have a number
+   of planned features lined up already.
+
+3. The first packages we offer for download through the new project home
+   page - users won't need to have GitHub account to access our builds 
+   any more.
+
+There's no human-readable changelog just yet - there are [~1500 commits][diff]
+to comb through. Hopefully, within a couple of days, we'll be able to write
+a nice summary with a description of new features, improvements, and bugfixes.
+
+In the meantime, we invite everyone to start testing RC1 builds; there are
+some rough edges for sure - if you find any bugs, please report them in
+[our bugtracker][bugtracker]. You can also ask questions on the
+**#dosbox&#8209;staging** channel in [Luxtorpeda Discord Server][discord].
+
+This release includes all upstream changes until SVN r4336, and 857 Git commits
+on top of that.
```

Comment:
Just trying to sort out the numbers into buckets: ~1500 total commits; we have upstream and then 857 of our own - does that mean upstream did ~643 commits in the same timeframe?

Reply:
Worse, upstream made that many commits to SVN over *last 10 years*:

```
$ git log --oneline svn/RELEASE_0_74..svn/trunk  | wc -l
689
```
That tag is 0.74 release; many of those 689 commits were backported to 0.74-2 and 0.74-3, but much fewer than I expected.